src/NuGet.Clients/NuGet.PackageManagement.UI/Resources.resx

```diff
@@ -999,4 +999,17 @@ Please see https://aka.ms/troubleshoot_nuget_cache for more help.</value>
     <value>You have {0} vulnerable package(s) installed.</value>
     <comment>{0} is the number of installed packages</comment>
   </data>
+  <data name="Deprecation_MoreInfo" xml:space="preserve">
+    <value>More info</value>
+  </data>
+  <data name="Deprecation_LinkTooltip" xml:space="preserve">
+    <value>Select to view the alternative package.</value>
+  </data>
+  <data name="Deprecation_PackageItemMessage" xml:space="preserve">
+    <value>The package is deprecated.</value>
+  </data>
+  <data name="Deprecation_PackageItemMessageAlternative" xml:space="preserve">
+    <value>The package is deprecated. Use {0} instead.</value>
```

Comment:
``` suggestion
    <value>The package version is deprecated. Use {0} instead.</value>
```

The reason is for https://www.nuget.org/packages/PeterO.Cbor package, the latest version (including few previous versions) is not deprecated but previous versions are.

Reply:
Consider:
```suggestion
    <value>This package version is deprecated. Use {0} instead.</value>
```

@anangaur any opinion regarding "Use {0} instead" vs "Try {0} instead". "Use" may imply certainty on our part that they should be able to use the alternative package with little issue.